### PR TITLE
Component library code fix

### PR DESCRIPTION
**Pull request by SinchanaMS**
Changes made:
1. Made the site responsive.
2. Added page navigation links at the bottom of each page.
3. Fixed the hover functionality of the Header links and Sidebar links.

Please check any one of the HTML files for the updated changes - page nav links. The same has been copied across the rest of them. Media queries have been added in documentation.css and main.css files.

### CODE REVIEW

**Comment by apoorvshrimali** [comment]
Beautiful use of colours 
code looks pretty clean

**Comment by AagmanSuri** [comment]
code looks good to merge

**Comment by shruthipatil11** [comment]
Code is clean, well structured  and UI is great and good to merge

**Review by m26unkwn — APPROVED** [review]
1. UI is clean and beautiful
2.  Colors pallet is appealing.
3.  Buttons  are good specially the FAB.

File: stylesheets/documentation.css [hunk 1]
```diff
@@ -329,39 +363,97 @@ body::-webkit-scrollbar-thumb {
...
+    .header-links {
+        display: flex;
+        gap: 7px;
+        justify-content: flex-end;
+        margin-right: 0px;
+        margin-top: 1rem;
+        align-items: flex-end;
+        font-weight: 600;
+        font-size: 14px;
+    }
```
Comment (ShrutiLakhani) [inline stylesheets/documentation.css:458]: Code looks clean and crisp.

**Review by ShrutiLakhani — APPROVED** [review]
Great UI. Good job. Code looks good to merge

**Comment by SinchanaMS** [comment]
Thank you all for the review! Merging to dev.

**Pull request merged**